feature-utils/poly-analysis/src/context/poly-analysis.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { createContext, useContext, useEffect, useState } from "react";
+
+export const PolyAnalysisContext = createContext();
+
+export const PolyAnalysisProvider = ({
+    children,
+    parentContext,
+    subAnalyses,
+    analyzeFile,
+}) => {
+    const { account, files } = useContext(parentContext);
+    const [fileAnalysis, setFileAnalysis] = useState(null);
+
+    //When the account model is created in the importer Context, the analyses are triggered.
+    useEffect(() => {
+        if (!files?.[0]) return;
+        analyzeFile({
+            zipData: files[0],
+            dataAccount: account,
```

Comment:
Also, no checks here in `account`. If `files` can be null, maybe this one can, too.